lib/services/android-debug-service.ts

```diff
@@ -141,6 +136,11 @@ class AndroidDebugService implements IDebugService {
 				this.detachDebugger(packageName).wait();
 			} else if (this.$options.debugBrk) {
 				this.startAppWithDebugger(packageFile, packageName).wait();
+			} else {
+				this.startAppWithDebugger(packageFile, packageName).wait();
+				//TODO: Find different way to make sure that the app is started.
+				sleep(500);
+				this.attachDebugger(device.deviceInfo.identifier, packageName).wait();
```

Comment:
maybe I'm totally missing something here, but shouldn't `startAppWithDebugger` do this... Based on it's name it looks like that's what it should do and we do not have to attach debugger again?
What's the case here?